src/toil/batchSystems/abstractBatchSystem.py

```diff
@@ -180,6 +180,26 @@ def getUpdatedBatchJob(self, maxWait):
                  that were killed.
         """
         raise NotImplementedError()
+        
+    def getSchedulingHint(self):
+        """
+        Get a hint for the user about anything that might be going wrong in the
```

Comment:
Hints in CWL are resource requirements, so to avoid confusion, could this explicitly state somewhere in the first sentence that this is intended to be a log message?  Something like:

s/`Get a hint for the user`/`Get a log message string serving as a hint for the user`/g ?